sdk/azidentity/chained_token_credential.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 )
 
-// ChainedTokenCredential provides a TokenCredential implementation which chains multiple TokenCredential implementations to be tried in order
-// until one of the GetToken methods returns a non-default AccessToken.
+// ChainedTokenCredential provides a TokenCredential implementation that chains multiple TokenCredential implementations to be tried in order
+// until one of the GetToken methods returns a non-nil AccessToken.
```

Comment:
"non-nil" and "non-default" are both really implementation details of success.  I'd word it more like `It returns the token from the first successful call to GetToken().`

Reply:
Done